deploy/playbooks/03_app.yml

```diff
@@ -41,10 +49,25 @@
       when: not env_file.stat.exists
 
     - name: Start docker compose to see if everything is running
-      shell: "docker compose up -d"
+      ansible.builtin.command:
+        chdir: "{{ ansible_user_dir }}"
+        cmd: "docker compose up -d"
+      register: output
+      changed_when: output.rc != 0
+      failed_when: output.rc != 0
```

Comment:
This does not make sense. How should ansible know if this has failed or changed if the conditional is the same?
For ansible:
changed = continue
failed = stop here

Reply:
Same same - it's just a linter thing